### ISSUE TITLE
Scenario Ontology: Link to OpenDRIVE map

### ISSUE THREAD

**Issue by jdsika** [GAIA-X4PLC-AAD/ontology-management-base#47]
I am looking at the following element:
```
"scenario:trafficSpace": {
      "@type": "general:Link",
      "general:url": {
        "@value": "https://inavlid.link.com/my_map.xodr",
        "@type": "xsd:anyURI"
      },
      "general:type": "Document"
    }
```

I think a map is its own asset with its own meta data description. In this case we would depend with a scenario on a map that has already been instantiated as a separate asset and should be referenced through the asset id?

Maybe @lenasauermann has an opinion here as well?

**Comment by heuerfin** [comment]
Yes, we thought of that too. And we would also prefer to use the DID as a link. However, we weren't sure if `xsd:anyURI` is the correct type for a DID. 
According to https://www.w3.org/TR/did-core/ "[DIDs](https://www.w3.org/TR/did-core/#dfn-decentralized-identifiers) are [URIs](https://www.w3.org/TR/did-core/#dfn-uri) that associate a [DID subject](https://www.w3.org/TR/did-core/#dfn-did-subjects) with a [DID document](https://www.w3.org/TR/did-core/#dfn-did-documents) allowing trustable interactions associated with that subject." so the `xsd:anyURI` would be probably appropriate. 

According to [https://www.ietf.org/rfc/rfc2396.txt](https://www.ietf.org/rfc/rfc2396.txt) I found the following regex for URI `^(([^:/?#]+):)?(//([^/?#]*))?([^?#]*)(\?([^#]*))?(#(.*))?`.
Using https://www.regextester.com/ and the sample DID `did:web:registry.gaia-x.eu:ExampleServiceOffering:tbkmzxk9LQZ-6mfDovXUPycG7hpE639RFRFs` this matches.

From this I would conclude that this is okay then? Are there any other opinions? Othewise I would change it and create a pull request.

**Comment by jdsika** [comment]
@robertschubert what is your opinion? It makes sense to me. What did method should we use for assets?

**Comment by heuerfin** [comment]
I just talked wit @lenasauermann about it. I'll create a new branch and use a DID.

**Comment by jdsika** [comment]
The other question is if we want to define "asset bundles" in a generic way. So remove these nested links and remove a separate ontology for a bundle?

**Comment by heuerfin** [comment]
I'm not sure about that one. Last time we discussed this, we concluded that we will not use bundles inside our ontology. The ontology however, should offer the oppertunity to modell required and optional "dependencies" or links. A data provider/marketplace can then offer a bundle based on these dependencies.

However, we can reopen that discussion.

**Comment by jdsika** [comment]
I am ok if you discussed it!

**Comment by jdsika** [comment]
@lenasauermann I think you missed the change here: https://github.com/GAIA-X4PLC-AAD/ontology-management-base/blob/main/scenario/scenario_shacl.ttl

**Comment by heuerfin** [comment]
You mean the example in there? https://github.com/GAIA-X4PLC-AAD/ontology-management-base/blob/main/scenario/scenario_shacl.ttl#L207C23-L207C54

**Comment by lenasauermann** [comment]
@heuerfin @jdsika Is this done?
